src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -543,6 +672,33 @@ std::string ContextAttribute::toString(void)
   return name;
 }
 
+/* ****************************************************************************
+*
+* toStringValue -
+*/
+std::string ContextAttribute::toStringValue(void)
+{
+  switch (valueType)
+  {
+  case ValueTypeString:
+    return stringValue;
+    break;
+
+  case ValueTypeNumber:
+    return "<double>";
+    // FIXME P10: return std::string(numberValue);
```

Comment:
This is the kind of stuff so easy to do in modern languages (Python, etc.) and so difficult in C/C++ :P

@kzangeli , could you implement this, please?